.github/workflows/tests-and-checks.yml

```diff
@@ -28,17 +28,17 @@ jobs:
 
       - name: Lint with Ruff
         run: |
-          ruff --output-format=github .
+          poetry run ruff --output-format=github .
```

Comment:
pls, fix warn
`warning: `ruff <path>` is deprecated. Use `ruff check <path>` instead.`

Reply:
Done